src/structures/aligned_segment.cpp

```diff
@@ -0,0 +1,25 @@
+#include "structures/aligned_segment.hpp"
+
+#include <seqan3/alphabet/cigar/cigar.hpp>
+
+#include "structures/breakend.hpp"          // for strand
```

Comment:
you do not use the `strand` in this file... include can be removed (+ it is anyway included in the corresponding header file)